R/LSBuffonsneedlemanipulation.R

```diff
@@ -0,0 +1,197 @@
+#
+# Copyright (C) 2019 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+LSBuffonsneedlemanipulation   <- function(jaspResults, dataset, options, state = NULL){
+  .buffonsNeedleManipulationSummaryTable(jaspResults, options)
+  .buffonsNeedleManipulationPropDistPlot(jaspResults, options)
+  .buffonsNeedleManipulationPiDistPlot(jaspResults, options)
+  return()
+}
+
+
+
+.buffonsNeedleManipulationSummaryTable <- function(jaspResults, options){
+  if(!is.null(jaspResults[["summaryTable"]])) return()
+  # example d for computation
+  d <- 5
+  l <- options[["length"]]*d/100
+  
+  ## Summary Table
+  summaryTable <- createJaspTable(title = gettext("Summary Table"))
+  summaryTable$position <- 1
+  summaryTable$dependOn(c("k", "n", "a", "b", "length", "CI"))
+  #summaryTable$addCitation("JASP Team (2018). JASP (Version 0.9.2) [Computer software].")
+  
+  summaryTable$addColumnInfo(name = "NumCrosses", title = gettext("Crosses"), type = "integer")
+  summaryTable$addColumnInfo(name = "NumObservations", title = gettext("Observations"), type = "integer")
+  summaryTable$addColumnInfo(name = "Median", title = gettextf("Median for %s", "\u03c0"),   type = "number")
+  summaryTable$addColumnInfo(name = "lowerCI", title = gettext("Lower"), type = "number", 
+                            overtitle = gettextf("%s%% Credible Interval", options[["CI"]]*100))
+  summaryTable$addColumnInfo(name = "upperCI", title = gettext("Upper"), type = "number", 
+                            overtitle = gettextf("%s%% Credible Interval", options[["CI"]]*100)) 
+
+
+  # fill in the table
+  CI95lower <- 2 * l / (qbeta((1-options[["CI"]])/2, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE) * d)
+  CI95lower = round(CI95lower, digit = 2)
```

Comment:
Generally, we stick to <- for variable assignments
```suggestion
  CI95lower <- round(CI95lower, digit = 2)
```

Reply:
This appears in multiple places